init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
+
+Write-Host "Installing Build Tools CLI version..."
+if (!(Test-Path "$DOTNET_PATH"))
+{
+     mkdir "$DOTNET_PATH"
+}
+
+$DOTNET_VERSION = Get-Content "$RepoRoot\BuildToolsCLIVersion.txt"
+$DOTNET_ZIP_NAME="dotnet-dev-win-x64." + $DOTNET_VERSION + ".zip"
```

Comment:
a link for this:
https://github.com/dotnet/core-setup/blob/release/1.0.0/build_projects/dotnet-host-build/build.ps1#L81

/cc @eerhardt Maybe we should make preview2 work on the new install scripts